services/director/src/simcore_service_director/producer.py

```diff
@@ -817,7 +822,15 @@ async def _get_service_key_version_from_docker_service(
         )
 
     service_full_name = service_full_name[len(config.REGISTRY_PATH) :].strip("/")
-    return service_full_name.split(":")[0], service_full_name.split(":")[1]
+    service_re_match = _SERVICE_KEY_REGEX.match(service_full_name)
+    if not service_re_match:
+        raise exceptions.DirectorException(
+            msg=f"service {service_full_name} does not contain valid simcore key"
+        )
+    service_key = service_re_match.group(1)
+    service_tag = service_re_match.group(4)
+    # FIXME: service_key:service_tag@sha256:230498230492834
```

Comment:
- remove FIXME comment
- add what a "valid simcore key" means in the exception message e.g. 
```
f"'{service_full_name}' in an invalid service name, expected name as 'simcore/services/comp/myservice:1.2.3' "
```